### PR TITLE
INJIMOB-978 - Update workflow to support js and kotlin publish

### PR DESCRIPTION
**Pull request by adityankannan-tw**
- Added npm publish job to workflow.

### CODE REVIEW

File: .github/workflows/publish-maven-npm.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Artifacts Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      message:
+        description: 'Message for manually triggering'
+        required: false
+        default: 'Triggered for inji-vc-client Updates'
```
Comment (KiruthikaJeyashankar) [inline .github/workflows/publish-maven-npm.yml:9]: inji-vci-client instead of inji-vc-client

**Review by KiruthikaJeyashankar — APPROVED** [review]

**Review by KiruthikaJeyashankar — CHANGES_REQUESTED** [review]
inji-vci-client instead of inji-vc-client

**Review by KiruthikaJeyashankar — APPROVED** [review]

**Comment by adityankannan-tw** [comment]
> inji-vci-client instead of inji-vc-client

done

**Review by KiruthikaJeyashankar — APPROVED** [review]

File: .github/workflows/publish-maven-npm.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_INJI_TEAM }}
+  
+  publish-to-npm-registry:
+    if: ${{ inputs.publish == 'Both[Kotlin and JS]' || inputs.publish == 'js'}}
+    uses: mosip/kattu/.github/workflows/npm-publish-to-npm-registry.yml@master
+    with:
+      SERVICE_LOCATION: 'js'
+    secrets:
+      NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
```
Comment (swatigoel) [inline .github/workflows/publish-maven-npm.yml:44]: secret should be SLACK_WEBHOOK_INJI_TEAM

**Review by swatigoel — CHANGES_REQUESTED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**